api/src/neo4j/queries/compartment.js

```diff
@@ -1,40 +1,42 @@
 import querySingleResult from 'neo4j/queryHandlers/single';
 
-const getCompartment = async ({ id, version, full }) => {
-  const v = version;
+const getCompartment = async ({ id, model, version, full }) => {
+  const m = model ? `:${model}` : '';
+  const v = version ? `:V${version}` : '';
```

Comment:
For https://github.com/MetabolicAtlas/MetabolicAtlas/pull/508#discussion_r481799695:

Good point, it was bothering me too but I was thinking it's nice to clearly see exactly how their values in every function. But I don't think that should weigh over the advantage of having it defined in a single place. Will try to add a utility function for this and see how that feels.